packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/__tests__/actions.spec.ts

```diff
@@ -334,13 +341,7 @@ describe('DevWorkspace store, actions', () => {
   });
 
   describe('startWorkspace', () => {
-    it('should create REQUEST_DEVWORKSPACE and UPDATE_DEVWORKSPACE when starting DevWorkspace', async () => {
-      const devWorkspace = new DevWorkspaceBuilder().build();
-
-      (checkRunningWorkspacesLimit as jest.Mock).mockImplementation(() => undefined);
-
-      (fetchServerConfig as jest.Mock).mockResolvedValueOnce({});
-
+    const prepareDevWorkspaceMocks = (devWorkspace: devfileApi.DevWorkspace) => {
```

Comment:
I prefer to do such a setup within the `beforeEach`, do you mind?

Reply:
Ok. I have fixed it.